app/scripts/lib/auth-errors.js

```diff
@@ -151,10 +151,6 @@ function (_, Errors) {
       errno: 1011,
       message: t('Valid email required')
     },
-    YEAR_OF_BIRTH_REQUIRED: {
```

Comment:
We spoke at one point about how we were going to handle deprecated client-side errors. The resolution was to comment out the errors and leave them in the code. This is usually considered bad practice with version control, but is reasonable from a documentation perspective and ensuring somebody doesn't come along and try to fill in one of the newly empty numbers.